libnmstate/nm/active_connection.py

```diff
@@ -108,13 +105,10 @@ def _deactivate_connection_callback(self, src_object, result, user_data):
 
         try:
             success = src_object.deactivate_connection_finish(result)
-        except Exception as e:
-            if (
-                isinstance(e, GLib.GError)
-                # pylint: disable=no-member
-                and e.domain == NM_MANAGER_ERROR_DOMAIN
-                and e.code == NM.ManagerError.CONNECTIONNOTACTIVE
-                # pylint: enable=no-member
+        except common.GLib.Error as e:
```

Comment:
What should happen if a different exception than `common.GLib.Error` is raised? Does this never happen?

Reply:
According to libnm documentation it always raises GLib.Error exception. https://lazka.github.io/pgi-docs/#NM-1.0/classes/Client.html#NM.Client.deactivate_connection_finish